index.html

```diff
@@ -230,25 +252,38 @@ <h2 class="about-card-title">Skills</h2>
                         <li><a href="#">Terminal</a></li>
                     </ul>
                 </div>
-            </div>
-        </article>
+            </article>
+        </div>
     </section>
 
     <section id="contact-frm" class="c-posts">
-        <article class="c-posts__item pb-0">
+        <article class="c-posts__item pb-0 c-form">
             <h1 class="section-title mt-0">
                 I'm always interested in hearing about new projects, so if you'd like
                 to chat please get in touch.
             </h1>
```

Comment:
- [ ] [Optional] I would suggest you please stick to the design as much as possible. If you see the contact me section then you will find that the content covers 40% of the width and the form cover the remaining 60% of the width. Please implement it.
### Your design
![his](https://user-images.githubusercontent.com/81727988/214856548-24f4ba8c-05b4-47da-9549-2c202e34370c.jpg)
### Figma design

![figma](https://user-images.githubusercontent.com/81727988/214856687-18cf7084-68dd-4730-b0ec-33b91fb4a1c3.jpg)